app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= link_to post.content, post %></p>
+<% if post.image_link != "" && !post.image_link.nil? %>
```

Comment:
- [ ] Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.

Reply:
Hi. Thanks for the feedback. We have moved the logic into the view helper file